lib/SIL/Linker.cpp

```diff
@@ -353,6 +353,10 @@ void SILLinkerVisitor::visitAllocRefInst(AllocRefInst *ARI) {
   linkInVTable(D);
 }
 
+void SILLinkerVisitor::visitReallocRefInst(ReallocRefInst *ARI) {
```

Comment:
Can you follow the model of AllocRefInst above. Generally linker changes like this are done in the instruction adding commit.